i18n/en.pot

```diff
@@ -38,11 +38,6 @@ msgstr "Created {{time}} by {{author}}"
 msgid "Created {{time}}"
 msgstr "Created {{time}}"
 
-msgid "Viewed {{count}} times"
```

Comment:
I don't see where these strings were removed in this PR.
Was it perhaps done in another PR but the pot file was not updated?

Reply:
Looks like the issue where i18n changes are not reflected in the pot files before yarn start or build have been run. So quite often the pot file is updated in the next PR. Not great.